app/apollo/resolvers/cluster.js

```diff
@@ -469,8 +469,8 @@ const clusterResolvers = {
         // Allow graphQL plugins to retrieve more information. deleteClusters can delete clusters. Include details of each deleted resource in pluginContext.
         const clusters = await commonClusterSearch(models, {org_id}, { limit: 0, skip: 0, startingAfter: null });
         context.pluginContext = {
-          clusters: clusters.map( c => {
-            return( { name: c.registration.name, uuid: c.cluster_id } );
+          deleted_clusters: clusters.map( c => {
+            return( {deleted_cluster: {registration_details: c.registration, uuid: c.cluster_id }} );
```

Comment:
- Just use `clusters` and and an object representing the cluster -- the code consuming the plugin context _knows_ this is a cluster delete mutation, there's no need to have `delete_` specific attribute names.
- cluster objects should have name and uuid at minimum -- full registration details can be separate, but `name` should be there even if the value comes from registration details.
```suggestion
          clusters: clusters.map( c => {
            return( { name: c.registration.name, uuid: c.cluster_id, registration: c.registration }} );
```